docs/stencil-docs/localization/localizing-stores.md

```diff
@@ -31,10 +31,27 @@ A BigCommerce storefront can be customized to display in any one language of you
 </div>
 </div>
 
-## Translating theme files via Stencil CLI
+## Localizing theme files
+
+### Translating theme files by Stencil CLI
 
 The bulk of localizing a store is translating content on your theme's page templates. You will need to abstract these templates' literal text strings into `{{lang}}` variables, then provide per-language JSON files that define these variables in corresponding key/value pairs.
 
+### Translating strings by BigCommerce
+
+Some of the strings displayed on your storefront are not currently editable in BigCommerce. These are strings such as "This promotion cannot be applied with the selected currency." For strings such as this BigCommerce, will automatically render the strings in the shopper's language as defined by the shopper's browser language setting.
+

```

Comment:
```suggestion
It may be best to list the languages alphabetically.

* Dutch
* English
* French
* German
* Italian
* Portuguese
* Spanish

```